Monika After Story/game/script-topics.rpy

```diff
@@ -17952,3 +17952,35 @@ label monika_overton:
     m 1eua "I {i}could{/i} finish this story right up to Current Norm, but I just wanted to describe it at a basic level in order to convey an example of how it can work."
     m 1huu "Thank you for listening~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_mercury",
+            category=["life"],
+            prompt="Mercury's orbit",
+            random=True
+        )
+    )
+
+label monika_overton:
+    m "Apparently, for a long while the astronomers believed that the planet Mercury is always facing the Sun with the same side."
+    m "In truth, it completes one and a half rotations in one orbital period...{w=0.1}{nw}"
+    extend "which makes the daylight cycle two orbital periods long."
+    m "Imagine a day lasting for two years!"
+    m "Anyway, {w=0.2}{nw}"
+    extend "the biggest issue with observation of Mercury is its closeness to the Sun."
```

Comment:
`    extend 4ruc "the biggest issue with observation of Mercury is its closeness to the Sun."`